### ISSUE TITLE
Agent is not picking up metadata on my Mac Server

### ISSUE THREAD

**Issue by eyung71** [LGBT-PlexPlugins/plex-gay-metadata-agent#50]
What version of Plex Gay Metadata Agent are you reporting this for?
I don't know what version of the Gay Metadata Agent but it's the one that is posted on this site as of Dec 24, 2018. I don't know where to find the version number.

Describe the bug A clear and concise description of what the bug is What section does it affect?
The plugin was installed as per the instructions for a Mac. I can see the Gay Adult agent and it is selected in my folder as the agent. I have 1200 files. I have scanned the library and refreshed all metadata. So far only 5 files of the 1200 files pick up metadata (2 of 73 Treasure Island titles are picked up, 3 out of 29 Helix Titles are picked up). All others pick up none. Most files show an 86 (%?) match under the info. But no descriptions, actors, poster or background images come down. Images are generated thumbs from the files themselves. Files are named to match the instructions.

Let me know what other info I need to give you to help sort this out. Appreciate the help! Would love to have this working!

To Reproduce Steps to reproduce the behavior:

Go to Library called 'Porn'
Click on Scan Library or Refresh all Metadata, wait 10-15 minutes until scan is complete
Scroll down to individual files...
See error
Files have generated thumbs and no metadata 
- uploaded screen shot example of SeanCody file that has 86 Match but no data. 
<img width="730" alt="seancody doesn t work file info 4" src="https://user-images.githubusercontent.com/45474441/50390185-f33a6c00-076e-11e9-9c14-33e5cacbf97b.png">
<img width="731" alt="seancody doesn t work file info 3" src="https://user-images.githubusercontent.com/45474441/50390186-f33a6c00-076e-11e9-825a-40673345d099.png">
<img width="732" alt="seancody doesn t work file info 2" src="https://user-images.githubusercontent.com/45474441/50390187-f3d30280-076e-11e9-8b3b-2b5b90cfb2a1.png">
<img width="1194" alt="seancody doesn t work file info 1" src="https://user-images.githubusercontent.com/45474441/50390188-f3d30280-076e-11e9-903f-d6b425a21ebe.png">

- uploaded screen shot example of TreasureIsland file what has 86 Match but does work.
<img width="728" alt="tim works file info 3" src="https://user-images.githubusercontent.com/45474441/50390173-cc7c3580-076e-11e9-817c-bb2caa185c6d.png">
<img width="731" alt="tim works file info 2" src="https://user-images.githubusercontent.com/45474441/50390174-cc7c3580-076e-11e9-82e9-6d922da87d6e.png">
<img width="1176" alt="tim works file info 1" src="https://user-images.githubusercontent.com/45474441/50390175-cd14cc00-076e-11e9-8916-2c99aba146a7.png">

Expected behavior A clear and concise description of what you expected to happen.

Screenshots / Screen recordings If applicable, add screenshots or recordings to help explain your problem. Please keep these short and if you can, edit them to point out when/where the problem is. https://www.useloom.com/ is perfect for screen recording with Chrome or with their desktop app

Server / Test environment (please complete the following information):

Local development? Deployed to a web server? 
I am running Plex Media Server on a Mac Mini. Files are on a Network Attached Storage on the same network as the mini. I'm not sure if answers this detail.
Operating system Mac OS 10.13.6 (High Sierra)
Plex version 3.77.2
Browser(s) tested with [e.g. chrome, safari - if applicable] Safari version 12.0 Chrome Version 69.0.3497.100
<img width="580" alt="gay adult metadata agent settings seancody" src="https://user-images.githubusercontent.com/45474441/50390123-fa14af00-076d-11e9-9cb1-4ae0dc8ca406.png">
<img width="845" alt="gay adult metadata agent settings" src="https://user-images.githubusercontent.com/45474441/50390124-fa14af00-076d-11e9-9667-a94095975bff.png">

Additional context Add any other context about the problem here.

**Comment by eyung71** [comment]
Ah I figured out what version I'm using. Its the 2017.08.03 version which is the most recent.

**Comment by MrPlow254** [comment]
Sorry, I don't get notifications. Thank you for uploading all the screenshots. Please also upload the log file after you do a meta data refresh. To do this on the last photo in hte Gay Adult  click the cog wheel for sean cody. enable logging. Then go to the log folder that plex saves do and upload the `com.plexapp.agents.seancody.log` file

Example path that windows uses is
C:\Users\{username}\AppData\Local\Plex Media Server\Logs\PMS Plugin Logs

**Comment by MrPlow254** [comment]
Actually, looking at their site it looks like it was changed from when the script was written. It is now broken for me as well. I will see what I can do, but there needs to be some universal site to use that doesn't break all the time. GayPornCollector was going to be that site but does not exist anymore.

**Comment by eyung71** [comment]
Thanks for the feedback! I appreciate the help. At least I know now that I've set it up correctly. I kept wondering if I'd missed something. 

Have you looked at this master database as a place to go? https://www.gayeroticvideoindex.com You're right that the studio sites and they have a habit of redesigning their sites every year or so and possibly even do this deliberately.

I'll get the SeanCody log in this later on today.
